src/components/ADempiere/Form/ProductSearch/index.vue

```diff
@@ -341,8 +394,86 @@ export default defineComponent({
       })
     }
 
+    function cleanQueryCriteria() {
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'product_category_id',
+        value: undefined
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'product_class_id',
+        value: undefined
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'product_classification_id',
+        value: undefined
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'product_group_id',
+        value: undefined
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'price_list_version_id',
+        value: undefined
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'is_stocked',
+        value: undefined
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'warehouse_id',
+        value: undefined
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'vendor_id',
+        value: undefined
+      })
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'search_value',
+        value: ''
+      })
+      searchProduct()
+    }
+
     setQuery()
 
+    function setCurrent(row) {
+      productInfoTable.value.setCurrentRow(row)
+    }
+
+    function close() {
+      closeTagView()
+    }
+
+    function setContextValue() {
+      store.commit('setProductSearchFieldQueryFilterByAttribute', {
+        containerUuid: uuidForm,
+        attributeKey: 'warehouse_id',
+        value: getContext({
```

Comment:
Please validate if context contains this value and value is `> 0` to mutate vuex state.

Reply:
Resolved in the following commit https://github.com/solop-develop/frontend-core/pull/2213/commits/da5650132e42f856e8def93a45a3cb4248112bf0